File: aoj/volume0/n0046/Main.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);
        double min = sc.nextDouble();
        double max = min;

        while (sc.hasNextDouble()) {

            double height = sc.nextDouble();

            if (height > max) {
                max = height;
            }

            if (height < min) {
                min = height;
            }

        }

        System.out.println(max - min);

    }

}

File: aoj/volume0/n0046/Main.cpp
#include <iostream>

using namespace std;

int main() {

    float min, max;
    cin >> min;
    max = min;

    float height;

    while (cin >> height) {

        if (height < min) {
            min = height;
        }

        if (height > max) {
            max = height;
        }

    }

    cout << max - min << endl;

    return 0;

}
